src/main/sensors/voltage.h

```diff
@@ -54,7 +55,7 @@ typedef enum {
 #define VBAT_SCALE_MIN 0
 #define VBAT_SCALE_MAX 255
 
-#define VBATT_LPF_FREQ  1.0f
```

Comment:
Is this change tested? 10x/4x reduction of filter time constant can surprise some users ...